Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -97,18 +97,22 @@ open class PassiveLocationManager: NSObject {
     public weak var delegate: PassiveLocationManagerDelegate?
     
     /**
-     Starts the generation of location updates. 
+     Starts the generation of location updates and resumes trip session.
+
+     To learn more read `PassiveLocationManager.resumeTripSession()` method documentation.
      */
+    @available(*, deprecated, message: "Use resumeTripSession() instead.")
     public func startUpdatingLocation() {
-        systemLocationManager.startUpdatingLocation()
```

Comment:
What if I call this method directly?

Reply:
You are on you own. You can still use `pauseTripSession`.